src/app/models/vehicle-tech-record.model.ts

```diff
@@ -244,6 +242,27 @@ export interface TechRecordModel {
   // Train weights
   maxTrainGbWeight?: number;
   trainDesignWeight?: number;
+
+  // Dimensions
```

Comment:
Its just a nit could keep dimensions but remove the others, doesn't stop merge how it is though

Reply:
I will leave it as is for now. We will address the types in the UI project shortly anyway. Right now this one `interface` spans over 70 lines of code, plus all the extra `enum`s and other small types scattered around the file.

This will be a valid discussion when we decide to refactor these and how we do it.